packages/react-fiche-service-public/src/components/Table.js

```diff
@@ -40,14 +48,20 @@ class Table extends React.PureComponent {
         <tbody>
           {rows.map((tr, index) => (
             <tr key={index}>
-              {tr.$.map((td, index) => (
-                <td key={index}>
-                  <ElementBuilder
-                    data={ignoreParagraph(td.$)}
-                    headingLevel={headingLevel + 1}
-                  />
-                </td>
-              ))}
+              {tr.$.map((td, index) => {
```

Comment:
`index` should be named `column`

Reply:
But it is simply and index ? the column is the "td"